test/extended/operators/olm.go

```diff
@@ -226,4 +226,28 @@ var _ = g.Describe("[Feature:Platform] an end user use OLM", func() {
 		}
 
 	})
+
+	// OCP-24818 - Checking OLM descriptors
+	// author: tbuskey@redhat.com
+	g.It("Checking OLM descriptors", func() {
+		olmErr := 0
+		olmErrDescriptor := []string{""}
+		olmExplains := []string{"InstallPlan", "ClusterServiceVersion", "Subscription", "CatalogSource", "OperatorSource", "OperatorGroup", "PackageManifest", "CatalogSourceConfig"}
```

Comment:
CatalogSourceConfig is not an OLM resource, it belongs to the Marketplace Operator.